sdk/communication/Azure.Communication.CallAutomation/src/CallMedia.cs

```diff
@@ -409,17 +409,21 @@ public virtual Response StopContinuousDtmfRecognition(CommunicationIdentifier ta
         /// <summary>
         /// Send Dtmf tones in async mode.
         /// </summary>
-        /// <param name="sendDtmfOptions">Configuration attributes for SendDtmf.</param>
+        /// <param name="targetParticipant">Target participants for start continuous Dtmf Recognition.</param>
+        /// <param name="tones">List of Tones to be sent.</param>
+        /// <param name="operationContext"></param>
         /// <param name="cancellationToken"></param>
         /// <returns></returns>
-        public virtual async Task<Response<SendDtmfResult>> SendDtmfAsync(SendDtmfOptions sendDtmfOptions,
-            CancellationToken cancellationToken = default)
+        public virtual async Task<Response<SendDtmfResult>> SendDtmfAsync(CommunicationIdentifier targetParticipant, IReadOnlyList<DtmfTone> tones,
```

Comment:
for input, we want IEnumerable<T> for parameters.

See comments from Azure Review Board:
> We prefer IEnumerable<T> for parameters and IList<T> or IReadOnlyList<T> for properties

Reply:
thank you, I have changed now.